### DISCUSSION TITLE
Dumber/

### DISCUSSION THREAD

**Discussion by giscus** [zmike/zmike.github.io#22] [Announcements]
# Dumber/

It’s Not Maintenance5

https://www.supergoodcode.com/Dumber/

**Comment by wispoffates** [comment]
Obscure ticket from years ago? Fixed ~~incidentally~~ 100% intentionally.

**Reply by zmike** [reply]
ye I'm playin the long game

**Comment by torokati44** [comment]
I love this garbage you shovel onto the internet.

**Reply by zmike** [reply]
good cuz I got a lot of it
